models/PartnerBillingParameters.v1.json

```diff
@@ -3,6 +3,13 @@
   "description": "Paramètres permettant de générer une facture à destination d'un partenaire.",
   "type": "object",
   "properties": {
+    "type": {
```

Comment:
I'm not sure we really need this.
We could already differentiate the two types by checking the existence of funding_results vs simulation?

Reply:
Yes, but the philosophy we used until then was to use a type to differentiate objects used in a oneOf.
You think we can do differently here ?